src/env.h

```diff
@@ -510,6 +530,7 @@ class Environment {
   AsyncHooks async_hooks_;
   DomainFlag domain_flag_;
   TickInfo tick_info_;
+  ArrayBufferAllocator array_buffer_allocator_;
```

Comment:
Let's call it `ArrayBufferAllocatorInfo` for gods sake :) I can imagine many people will be confused over this.